app/javascript/components/users/SignInPage.tsx

```diff
@@ -80,152 +78,149 @@ function SignInPage(_props:SignInPageProps) : JSX.Element {
         backgroundColor: "#3f51b5",
       },
       appbar: {
-        background: grey[200],
+        backgroundColor: theme.palette.action.hover, 
```

Comment:
Why `theme.palette.action.hover`? That theme color is the color to use related to hovering over a link, I think. Would people modifying the them expect that changing that the hover color would change the appbars on this page?

Reply:
We've changed this to be inherit.